### DISCUSSION TITLE
"File" dialog significant speed decrease from 2.6.0

### DISCUSSION THREAD

**Discussion by BruciusL** [artisan-roaster-scope/artisan#956] [Q&A] [answered]
Hello,
 I've got artisan running on an admittedly rather underpowered netbook on Arch Linux, but it's been more than adequate thus far.  Since 2.6.0, using save as, or open, gives a very long pause (45 seconds or so) before the file list is displayed.  If I just save the profile to the current name, or open from the recent list, it's the normal speed (a few seconds).  Going back to 2.4.6 resolves the problem.  
Has anything changed in the way artisan tries to list files, or is there anything I can do to help pinpoint the cause for this?
Thanks!

**Comment by MAKOMO** [comment, marked as answer]
The file selector handling is delegated by Artisan to the Qt framework (via PyQt) and executed by the OS. Nothing changed there on the Artisan side, however the Qt/PyQt versions changed. Artisan v2.4.6 runs on Qt/PyQt versions 5.15.25.15.5, while Artisan v2.6.0 runs on Qt/PyQt version 6.2.3/6.2.3. I fear not much can be done on the Artisan side to resolve this.

**Reply by BruciusL** [reply]
Thank you Marko.  I appreciate your quick and precise feedback.  I'll stick it out, possibly it improves with a future version of Qt/PyQt again!
